spec/factories/partners.rb

```diff
@@ -40,11 +40,20 @@
       status { :awaiting_review }
     end
 
+    transient do
+      pick_up_person { false }
```

Comment:
Considering this trait is only used once in the entire test suite, it seems like it doesn't need to exist, and the test can create the necessary data itself.